### ISSUE TITLE
Operation timed out

### ISSUE THREAD

**Issue by havvg** [keybase/client#6086]
I'm constantly getting `▶ ERROR Operation timed out`, log id: 5e430d75124d762c0440801c.

I'm trying to `keybase device add` for days since the keybase GUI has been released.

**Comment by maxtaco** [comment]
Are you running `login` and `device add` on the same device?

**Comment by havvg** [comment]
Yes, I am, as I only got this device.

**Comment by maxtaco** [comment]
Yeah, you're not supposed to do that.  You can only use `device add` on your first device if you're adding a second

cc: @patrickxb

**Comment by havvg** [comment]
How am I supposed to re-use my already added device? Since the change/switch to this GUI version it doesn't recognise my device. I can't remove the old one and I can't re-add the existing one using the paper key.

I would like to re-add the device with its original name, but that's not possible as long the "old" one exists - which is the very same device.

**Comment by maxtaco** [comment]
You have to pick a new name.
